modules/installing-extensions-with-code-server.adoc

```diff
@@ -0,0 +1,36 @@
+:_module-type: PROCEDURE
+
+[id="installing-extensions-with-code-server_{context}"]
+= Installing extensions with code-server
+
+With the code-server workbench image, you can customize your code-server environment by using extensions to add new languages, themes, and debuggers, and to connect to additional services. You can also enhance the efficiency of your data science work with extensions for syntax highlighting, auto-indentation, and bracket matching. 
+
+For details about the third-party extensions that you can install with code-server, see the link:https://open-vsx.org/[Open VSX Registry].
```

Comment:
The procedure to install extensions from VSX does not actually work ;(

https://github.com/eclipse/openvsx/wiki/Using-Open-VSX-in-VS-Code mentions product.json, with help from https://github.com/flathub/com.vscodium.codium/issues/90 I found it at `/usr/lib/code-server/lib/vscode/product.json` but there's nothing about any extension galery specified in there.

The steps https://github.com/opendatahub-io/notebooks/blob/main/codeserver/Extensions.md I don't see any Resources -> Marketplace to navigate to.

Reply:
https://github.com/coder/code-server/blob/4af06de4c3757b4857068a671f7dba7af6fb9179/CHANGELOG.md?plain=1#L822C1-L823C1 says that OpenVSX should be the default marketplace

Reply:
Disregard this, I was not properly following the instructions.

I have to type something into the Search bar, it does not start listing available extensions until I either search for something, (or vscode decides to recommend me some extension based on edited file extension...)

I think I knew this before, but I forgot. And the docs do say I need to search, but I thought the search works more like a filter for what's already displayed, not that it searches in catalogue of things not seen before.